### PR TITLE
Finish Like and Dislike Function

### PR DESCRIPTION
**Pull request by Phanithism**

### CODE REVIEW

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -70,6 +80,6 @@ def set_game
 
     # Never trust parameters from the scary internet, only allow the white list through.
     def game_params
-      params.require(:game).permit(:title, :image, :description, :release_date, :iframe, :source, :category_id)
+      params.require(:game).permit(:title, :image, :description, :release_date, :iframe, :source, :Category_id)
```
Comment (jensendarren) [inline app/controllers/games_controller.rb]: Should be lower case category_id

File: app/views/games/_form.html.erb [hunk 2]
```diff
@@ -36,14 +36,12 @@
     <%= f.text_area :source %>
   </div>
   <div class="field">
-    <%= f.label :cat_name %><br>
-    <%= f.select(:category) do %>
-  <% @category.each do |c| -%>
-    <%= content_tag(:option, c.name, value: c) %>
-  <% end %>
-<% end %>
+    <%= f.label :Category_id %><br>
```
Comment (jensendarren) [inline app/views/games/_form.html.erb]: Lower case 'c' please

File: app/views/homepage/index.html.erb [hunk 3]
```diff
@@ -2,48 +2,38 @@
   <div class="panel-heading">Features</div>
   <div class="panel-body">
 
-    <% Game.all.limit(5).order('created_at DESC').each do |g| %>
-      <div class="row">
-
-        <div class="col-sm-6 col-md-3">
+    <% Game.all.order('created_at DESC').each do |g| %>
```
Comment (jensendarren) [inline app/views/homepage/index.html.erb]: This should be in the model as a scope and in the controller as an instance var.

File: app/views/games/_form.html.erb (hunk 2 above)
Comment (Phanithism) [inline app/views/games/_form.html.erb]: Ok I'm fixing it now.

File: app/views/layouts/_navbar.html.erb [hunk 4]
```diff
@@ -16,7 +16,7 @@
       <ul class="nav navbar-nav">
 
       <% Category.all.limit(5).each do |c| %>
-        <li><%= link_to c.name , '#'%></li>
+        <li><%= link_to c.Cate_name, '#'%></li>
```
Comment (jensendarren) [inline app/views/layouts/_navbar.html.erb]: please change to 'category_name'

File: config/routes.rb [hunk 5]
```diff
@@ -1,6 +1,11 @@
...
   devise_for :users
-  resources :games
+  resources :games do
+    member do
+      put 'like', to: 'games#like_game'
+      put 'dislike', to: 'games#dislike_game'
+    end
+  end
   root 'homepage#index'
   get 'games/index'
```
Comment (jensendarren) [inline config/routes.rb]: Do we need this 'games/index' route? It should be provided by resources :games already.

**Pull request merged**